openassistant/datasets/mt_note_generation/prepare.py

```diff
@@ -0,0 +1,78 @@
+import json
+import math
+import os
+import random
+import re
+import sys
+from string import punctuation
+
+import kaggle
+import pandas as pd
+
+CLINICAL_NOTE_GENERATION_TEMPLATE = """User: Write a clinical note about a patient with the following {section}: {section_information}.
+ASSISTANT: {note}"""
```

Comment:
Change to "Assistant" or "Rosey" please.

Reply:
done